internal/lm/nvml.go

```diff
@@ -193,3 +201,59 @@ func isMPSCapable(manager resource.Manager) (bool, error) {
 	}
 	return true, nil
 }
+
+func newGPUModeLabeler(manager resource.Manager) (Labeler, error) {
```

Comment:
Does it make sense to add a unit test for this labeler?

Reply:
Added